json/actions.json

```diff
@@ -120,6 +120,7 @@
   },
   "ludopatico": {
     "type": "luck",
+    "importantGroup": "Non puoi usare questo comando in quanto questo gruppo è ritentuo importante",
```

Comment:
Già che sistemi il refuso in "ritentuo" potresti rifrasare come "Questo gruppo è troppo importante per usare questo comando."